community/cypher/cypher-planner-3.4/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/TxLayerPlanningIntegrationTest.scala

```diff
@@ -27,60 +28,68 @@ import org.scalatest.matchers.{MatchResult, Matcher}
 class TxLayerPlanningIntegrationTest extends CypherFunSuite with LogicalPlanningTestSupport2 {
 
   test("MATCH and CREATE") {
-    val plan = planFor("MATCH (a) CREATE (b)")._2
+    val tuple = planFor("MATCH (a) CREATE (b)")
```

Comment:
Is it allowed to type these as `val (_, plan, _, readTxLayerAttribute) = planFor(...)`?